build/templates/run-service-bus-test-worker.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  buildVersion: ''
+  healthPort: ''
+
+steps:
+- bash: |
+    if [ -z "$BUILD_VERSION" ]; then
+      echo "##vso[task.logissue type=error;]Missing template parameter \"manualTriggerVersion\""
+      echo "##vso[task.complete result=Failed;]"
+    fi
+    if [ -z "$HEALTH_PORT" ]; then
+      echo "##vso[task.logissue type=error;]Missing template parameter \"healthPort\""
+      echo "##vso[task.complete result=Failed;]"
+    fi
+  env:
+    BUILD_VERSION: ${{ parameters.buildVersion }}
+    HEALTH_PORT: ${{ parameters.healthPort }}
+  displayName: Check for required parameters in YAML template
+- task: Docker@1
+  displayName: 'Build Docker image for Service Bus test worker'
+  inputs:
+    dockerFile: src/Arcus.Messaging.Tests.Worker/Dockerfile
+    imageName: 'Arcus.Messaging.Tests.Worker:${{ parameters.buildVersion }}'
+    useDefaultContext: false
+    buildContext: src
+- task: Docker@1
+  displayName: 'Run Service Bus test worker from Docker image'
+  inputs:
+    command: 'Run an image'
+    imageName: 'Arcus.Messaging.Tests.Worker:${{ parameters.buildVersion }}'
```

Comment:
May want to specify a project name also, since `Arcus.Messaging.Tests.Worker` is now used directly.

Reply:
Can you elaborate on that please?

Reply:
Ah, nevermind. I was thinking, if we would like to use this for other test worker projects; you may want to add the project name as a parameter for this template also.

Reply:
Ah I see, it's part of a template. Feel free to use `${{ parameters.imageName }}:${{ parameters.buildVersion }}` indeed, but assign `Arcus.Messaging.Tests.Worker` as default `imageName` maybe?

Reply:
or call it project name since we need it on https://github.com/arcus-azure/arcus.messaging/pull/31/files#diff-c236825ac480abf2959ebbefe69480dfR22

Reply:
Updated!